test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
hmmmm #2201 reminds me of older EmberJS code that heavily relied on state machines - which is what i think @jmeas is talking about. i agree that routes represent states. 

even if @thejameskyle's example isn't the "correct" way of managing sub-applications in a router, it gets the point across that what i have been calling a sub-application is probably going to be initialized very explicitly in code. even in cases where you are loading another .js resource to bring in more functionality, that code will likely expose an API that the loading code knows about. 

personally, I don't think @thejameskyle's example is bad, even under the description of routes as a state. in this case, the route happens to require a new Application for this particular state. As long as that is clearly stated in documentation and examples, then I don't think there should be an issue. 

so, ok. +1 for removing this behavior if this is the direction that Marionette is heading - to allow multiple Application objects, as needed. i'm assuming (requesting!!!!) that marionette's module system is also going away? please? :)